### PR TITLE
Deploy script

### PR DESCRIPTION
**Pull request by freeman91**
Story: [105](https://5911-capstone.atlassian.net/browse/OCA-105)

## Summary

- add backend deploy job
- add a script to manually deploy
- add a script to run on ec2 instance, update the code, restart services

### CODE REVIEW

File: script/restart-services.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    exit 1
+fi
+
+echo -e "${GREEN}Fetching latest master${NC}"
+cd OSUMC-Cultural-Awareness-App/
+git fetch
+git pull origin master
+
+echo -e "${GREEN}Installing python packages${NC}"
+/home/ec2-user/.local/bin/pipenv install
```
Comment (NickHackman) [inline script/restart-services.sh:26]: If we're using pipenv can we remove `requirements.txt`?

File: script/restart-services.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+if [ $1 == 0 ]
+  then
+    echo -e "${RED}provide an ssh key as an argument${NC}"
+    exit 1
+fi
+
+echo -e "${GREEN}Fetching latest master${NC}"
+cd OSUMC-Cultural-Awareness-App/
+git fetch
+git pull origin master
```
Comment (NickHackman) [inline script/restart-services.sh:23]: Isn't fetching then pulling redundant?

**Review by NickHackman — APPROVED** [review]
Couple questions

File: script/restart-services.sh (hunk 2 above)
Comment (freeman91) [reply script/restart-services.sh:23]: true

File: script/restart-services.sh (hunk 1 above)
Comment (freeman91) [reply script/restart-services.sh:26]: yeah I thought I already got rid of it

**Pull request merged**